lib/SparqlEndpointFetcher.ts

```diff
@@ -172,17 +174,22 @@ export class SparqlEndpointFetcher {
    */
   public async fetchRawStream(endpoint: string, query: string, acceptHeader: string)
     : Promise<[string, NodeJS.ReadableStream]> {
-    const url: string = this.method === 'POST' ? endpoint : endpoint + '?query=' + encodeURIComponent(query);
+    let url: string = this.method === 'POST' ? endpoint : endpoint + '?query=' + encodeURIComponent(query);
 
     // Initiate request
     const headers: Headers = new Headers();
-    let body: BodyInit | undefined;
+    let body: URLSearchParams | undefined;
     headers.append('Accept', acceptHeader);
     if (this.method === 'POST') {
       headers.append('Content-Type', 'application/x-www-form-urlencoded');
       body = new URLSearchParams();
       body.set('query', query);
+      this.additionalUrlParams.forEach((key: string, value: string) => {
+        body.set(key, String(value));
```

Comment:
Since `value` is a `string`, I assume this can become:
```suggestion
        body.set(key, value);
```
?

Reply:
It can indeed (didn't fully realize that in a non-typescript situation not only the source, yet also the target are more relaxed about what they accept).